src/Components/ScrivitoExtensions/ColumnsEditorTab.js

```diff
@@ -149,6 +153,19 @@ function ColumnsEditorTab({ widget }) {
   }
 }
 
+function calculateContents(widget) {
+  return widget.get("columns").map((column) => column.get("content"));
+}
+
+function calculateContentIds(contents) {
+  return flatten(contents.map((content) => content.map((o) => o.id()))).sort();
```

Comment:
* [ ] Flattening and sorting may lead to missed edits. If our goal is to prevent losing (i.e. overriding) data entered by another editor (here: the position and order of widgets), we need a better content equality model.